frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -47,9 +48,10 @@ export const UserAvatar: FC<IUserAvatarProps> = ({
     className,
     sx,
     children,
+    hideTitle,
     ...props
 }) => {
-    if (!title && !onMouseEnter && user) {
+    if (!hideTitle && !title && !onMouseEnter && user) {
```

Comment:
do we need both hideTitle and title? Can't we not pass the title to hide it?

Reply:
Not the way it is today, no. If you don't pass a title and don't pass a mouseover function, it'll create a title for you. I *could* pass a no-op mouseover function from the avatar cell, but that feels like you have to know too much about the internals to make the component work as you intended.

Reply:
That said, I'm happy for you to dispute that and take it from there.

Also, I don't want to modify how we do titles from before, because it might cause issues elsewhere.

But yeah, that component *is* a bit messy, and this doesn't really make it better 💁🏼

Reply:
Don't want to cause distractions to other work in this PR. Generally when I find myself adding boolean props to my components it means composition is needed (boolean prop is configuration which doesn't scale very well). To achieve composition over configuration I normally pass the actual component I want to compose with and the API surface is ReactNode instead of a boolean flag.